vagrant-spk

```diff
@@ -705,24 +739,26 @@ def main():
         ('publish', publish,
             "Send the current packed app spk to \n"
             "  the Sandstorm App Market."),
-        ('halt', halt,
-            "Shut down the Vagrant VM."),
-        ('destroy', destroy,
-            "Delete the Vagrant VM (but leave \n"
+        ('vm', vm_subcommand,
+            "Pass through all following arguments to `vagrant`."),
+        ('halt', deprecated_halt,
+            "(deprecated, use 'vm halt') Shut down the Vagrant VM."),
+        ('destroy', deprecated_destroy,
+            "(deprecated, use 'vm destroy') Delete the Vagrant VM (but leave \n"
             "  --work-directory untouched)."),
-        ('global-status', global_status,
-            "Show the state of all active \n"
+        ('global-status', deprecated_global_status,
+            "(deprecated, use 'vm global-status') Show the state of all active \n"
             "  Vagrant VMs on this host."),
-        ('ssh', ssh,
-            "SSH into the Vagrant VM."),
+        ('ssh', deprecated_ssh,
+            "(deprecated, use 'vm ssh') SSH into the Vagrant VM."),
         ('keygen', keygen,
             "Generate a new key, add it to your keyring,\n"
             "  and print the identifier."),
         ('getkey', getkey,
             "Given a key identifier, retrieve and print the\n"
             "  corresponding private key from your keyring."),
-        ('reload', do_reload,
-            "Reboot the virtual machine, turning it off then on."),
+        ('reload', deprecated_do_reload,
+            "(deprecated, use 'vm reload') Reboot the virtual machine, turning it off then on."),
```

Comment:
Hmm.  Maybe, but I'd really like it to be minimal.  The purpose of `vagrant-spk` (and its help options) is not to teach people how to vagrant.

Such a list should probably contain at most "up", "halt", and maybe "ssh".